android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/PwsClient.java

```diff
@@ -37,29 +37,33 @@
 import java.util.Collection;
 
 /**
- * Class for resolving url metadata.
- * Sends requests to the metadata server
- * which then scrapes the page at the given url
- * for its metadata
+ * This class sends requests to the physical web service.
+ *
+ * For the largest use, metadata requests, the physical web service
+ * scrapes the page at the given url for its metadata.
  */
-
-class MetadataResolver {
-  private static final String TAG = "MetadataResolver";
-  private static final String METADATA_URL = "http://url-caster.appspot.com/resolve-scan";
-  //private static final String METADATA_URL = "http://url-caster-dev.appspot.com/resolve-scan";
-  private static final String DEMO_METADATA_URL = "http://url-caster.appspot.com/demo";
+class PwsClient {
```

Comment:
May we scan the codebase for references to this old class.. I think it's used in some docs.